### PR TITLE
Build Header Bar

### PR DESCRIPTION
**Pull request by schowkekarats360**

### CODE REVIEW

File: assets/src/apps/authoring/HeaderNav.tsx [hunk 1]
```diff
@@ -7,8 +17,64 @@ interface HeaderNavProps {
 
 const HeaderNav: React.FC<HeaderNavProps> = (props: HeaderNavProps) => {
   const { content, isVisible } = props;
+  console.log(content);
```
Comment (dsuriano) [inline assets/src/apps/authoring/HeaderNav.tsx:20]: 🗑

File: assets/src/apps/authoring/HeaderNav.tsx [hunk 2]
```diff
@@ -7,8 +17,64 @@ interface HeaderNavProps {
...
+        <div className="btn-group" role="group">
+          <div className="pr-3 border-right">
+            INFINISCOPE{/* Place for Logo */}
+          </div>
+          <div className="pl-3">
+            {content.title}
+          </div>
+        </div>
+      </div>
+      <div className="col-7 btn-toolbar" role="toolbar" aria-label="Toolbar with button groups">
```
Comment (dsuriano) [inline assets/src/apps/authoring/HeaderNav.tsx:38]: i think you can safely delete the `col-*` from these divs that are children of the `<nav>`. they are causing the header to show scrollbars the moment the screen is resized...

**Review by dsuriano — CHANGES_REQUESTED** [review]

**Review by dsuriano — APPROVED** [review]
Tested locally and verified scrollbars fixed.

**Comment by schowkekarats360** [comment]
closing this one as latest PR is in place.

**Pull request closed without merging**